package.json

```diff
@@ -26,9 +25,8 @@
     "utility"
   ],
   "devDependencies": {
-    "benchmark": "2.1.4",
-    "browserify": "16.2.3",
-    "mocha": "5.2.0",
-    "opn-cli": "4.0.0"
+    "benchmark": "^2.1.4",
+    "browserify": "^16.2.3",
+    "mocha": "^8.3.2"
```

Comment:
Updated `mocha` too, the listed version actually threw errors on startup for me.